### ISSUE TITLE
Romance of the Three Kingdoms III - Dragon of Destiny (1993)

### ISSUE THREAD

**Issue by TimberlAndRE-de** [exoscoriae/eXoDOS#573]
looks like a beta manual  images are missing and there are notes in it saying insert imgage ... here
means there is conent missing

**Comment by SmilingSpectre** [comment]
It's not "beta", it's typed manual, saved as PDF.

I found SEGA and SNES manuals. And they are essentially the same, except some controls. I suggest to add SNES as extra, because at least it's _nice_, and functional is mostly the same. Uploaded.

**Comment by TimberlAndRE-de** [comment]
@exoscoriae true, the snes one looks nice ... but it's snes. i suggest to keep issue open and search for the real manual?

**Comment by TimberlAndRE-de** [comment]
since i changed my mind about other platform manuals, the snes one should be added ;-)
uploaded to the github folder

**Comment by exoscoriae** [comment]
keeping current. no reason to start putting console manuals in here, even if they are better quality.
